remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerImpl.java

```diff
@@ -120,6 +127,31 @@ private ConfigurationBuilder getProtobufMetadataCacheConfig() {
          globalConfiguration.security().authorization().roles().put(SCHEMA_MANAGER_ROLE, new CacheRoleImpl(SCHEMA_MANAGER_ROLE, AuthorizationPermission.ALL));
          cfg.security().authorization().enable().role(SCHEMA_MANAGER_ROLE);
       }
+
+      cfg = addMetadataSiteBackups(cfg, cacheManager);
+
+      return cfg;
+   }
+
+   private ConfigurationBuilder addMetadataSiteBackups(
+         ConfigurationBuilder cfg, EmbeddedCacheManager cacheManager) {
+      final List<String> allCacheNames = new ArrayList<>(cacheManager.getCacheConfigurationNames());
+      allCacheNames.add(BasicCacheContainer.DEFAULT_CACHE_NAME);
+
+      allCacheNames.stream()
```

Comment:
What happens when the remote site does not have protobuf cache configured? (is this possible?)